### ISSUE TITLE
Crashed after closing 3d map

### ISSUE THREAD

**Issue by RoscoP** [EDDiscovery/EDDiscovery#2458]
### Description
Was looking at the 3d map, panning around, turned off some items in the display, then closed it.  Then switched to the main EDDiscovery window and it crashed.

As a side note, I noticed my install has .pdb files included, might consider removing these since they shouldn't be needed and take up a significant amount of space.

### Additional Information
Here is the last part of the tracelog, the rest has my APIKEY and lots of http requests
```
[2019-03-23T22:29:22.530Z] EDSM Comments downloaded/updated 0
[2019-03-23T22:35:38.813Z] HTTPGET TO https://www.edsm.net/api-logs-v1/get-logs?showId=1&apiKey=APIKEY&commanderName=COMMANDERNAME&endDateTime=2019-03-23+22%3a35%3a38 ThreadEDSM Log Fetcher
[2019-03-23T22:35:39.586Z] ..HTTPGET Response OK
[2019-03-23T22:50:38.931Z] HTTPGET TO https://www.edsm.net/api-logs-v1/get-logs?showId=1&apiKey=APIKEY&commanderName=COMMANDERNAME&endDateTime=2019-03-23+22%3a50%3a38 ThreadEDSM Log Fetcher
[2019-03-23T22:50:39.445Z] ..HTTPGET Response OK
[2019-03-23T23:05:38.347Z] HTTPGET TO https://www.edsm.net/api-logs-v1/get-logs?showId=1&apiKey=APIKEY&commanderName=COMMANDERNAME&endDateTime=2019-03-23+23%3a05%3a38 ThreadEDSM Log Fetcher
[2019-03-23T23:05:38.897Z] ..HTTPGET Response OK
[2019-03-23T23:10:21.492Z] HTTPGET TO https://www.edsm.net/api-v1/systems?startdatetime=2019-03-23+22%3a05%3a47&enddatetime=2019-03-23+23%3a10%3a21&coords=1&submitted=1&known=1&showId=1 ThreadBackground Worker Thread
[2019-03-23T23:10:24.521Z] ..HTTPGET Response OK
[2019-03-23T23:20:38.035Z] HTTPGET TO https://www.edsm.net/api-logs-v1/get-logs?showId=1&apiKey=APIKEY&commanderName=COMMANDERNAME&endDateTime=2019-03-23+23%3a20%3a38 ThreadEDSM Log Fetcher
[2019-03-23T23:20:38.563Z] ..HTTPGET Response OK
[2019-03-23T23:29:23.316Z] HTTPGET TO https://www.edsm.net/api-logs-v1/get-comments?startdatetime=2011-01-01+00%3a00%3a00&apiKey=APIKEY&commanderName=COMMANDERNAME&showId=1 ThreadEDSM Log Fetcher
[2019-03-23T23:29:23.807Z] ..HTTPGET Response OK
[2019-03-23T23:29:23.807Z] EDSM Comments downloaded/updated 0
[2019-03-23T23:31:33.112Z] 911227049 Close form
[2019-03-23T23:31:33.602Z] 911227548 Ask for compute exit
[2019-03-23T23:31:33.602Z] 911227548 compute exit
[2019-03-23T23:31:36.329Z] 
==== UNHANDLED UI EXCEPTION ====
System.ArgumentException: Parameter is not valid.
   at System.Drawing.Bitmap..ctor(Int32 width, Int32 height, PixelFormat format)
   at BaseUtils.BitMapHelpers.DrawTextIntoAutoSizedBitmap(String text, Size maxsize, Font dp, Color c, Color b, Single backscale, StringFormat frmt) in C:\Code\EDDiscovery\BaseUtilities\BaseUtilities\WinForms\BitMapHelpers.cs:line 98
   at EDDiscovery.UserControls.UserControlSurveyor.DrawToScreen(WantedBodies body, Int32 bodiesCount) in C:\Code\EDDiscovery\EDDiscovery\UserControls\Overlays\UserControlSurveyor.cs:line 374
   at EDDiscovery.UserControls.UserControlSurveyor.SelectBodiesToDisplay(List`1 wanted_nodes) in C:\Code\EDDiscovery\EDDiscovery\UserControls\Overlays\UserControlSurveyor.cs:line 319
   at EDDiscovery.UserControls.UserControlSurveyor.DrawSystem(HistoryEntry he) in C:\Code\EDDiscovery\EDDiscovery\UserControls\Overlays\UserControlSurveyor.cs:line 277
   at System.Windows.Forms.Control.OnResize(EventArgs e)
   at System.Windows.Forms.UserControl.OnResize(EventArgs e)
   at System.Windows.Forms.Control.OnSizeChanged(EventArgs e)
   at System.Windows.Forms.Control.UpdateBounds(Int32 x, Int32 y, Int32 width, Int32 height, Int32 clientWidth, Int32 clientHeight)
   at System.Windows.Forms.Control.UpdateBounds()
   at System.Windows.Forms.Control.WmWindowPosChanged(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.UserControl.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
==== cut ====
[2019-03-23T23:35:38.172Z] HTTPGET TO https://www.edsm.net/api-logs-v1/get-logs?showId=1&apiKey=APIKEY&commanderName=COMMANDERNAME&endDateTime=2019-03-23+23%3a35%3a38 ThreadEDSM Log Fetcher
[2019-03-23T23:35:38.683Z] ..HTTPGET Response OK
```

### Exception Details:
>EDDiscovery v10.5.2.0 There was an unhandled UI exception.
>```
>==== BEGIN ====
>System.ArgumentException: Parameter is not valid.
>   at System.Drawing.Bitmap..ctor(Int32 width, Int32 height, PixelFormat format)
>   at BaseUtils.BitMapHelpers.DrawTextIntoAutoSizedBitmap(String text, Size maxsize, Font dp, Color c, Color b, Single backscale, StringFormat frmt) in C:\Code\EDDiscovery\BaseUtilities\BaseUtilities\WinForms\BitMapHelpers.cs:line 98
>   at EDDiscovery.UserControls.UserControlSurveyor.DrawToScreen(WantedBodies body, Int32 bodiesCount) in C:\Code\EDDiscovery\EDDiscovery\UserControls\Overlays\UserControlSurveyor.cs:line 374
>   at EDDiscovery.UserControls.UserControlSurveyor.SelectBodiesToDisplay(List`1 wanted_nodes) in C:\Code\EDDiscovery\EDDiscovery\UserControls\Overlays\UserControlSurveyor.cs:line 319
>   at EDDiscovery.UserControls.UserControlSurveyor.DrawSystem(HistoryEntry he) in C:\Code\EDDiscovery\EDDiscovery\UserControls\Overlays\UserControlSurveyor.cs:line 277
>   at System.Windows.Forms.Control.OnResize(EventArgs e)
>   at System.Windows.Forms.UserControl.OnResize(EventArgs e)
>   at System.Windows.Forms.Control.OnSizeChanged(EventArgs e)
>   at System.Windows.Forms.Control.UpdateBounds(Int32 x, Int32 y, Int32 width, Int32 height, Int32 clientWidth, Int32 clientHeight)
>   at System.Windows.Forms.Control.UpdateBounds()
>   at System.Windows.Forms.Control.WmWindowPosChanged(Message& m)
>   at System.Windows.Forms.Control.WndProc(Message& m)
>   at System.Windows.Forms.UserControl.WndProc(Message& m)
>   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
>===== END =====
>```

**Comment by klightspeed** [comment]
That looks like some icons were disposed when they shouldn't have.

**Comment by robbyxp1** [comment]
Same issue as previous, fixed in 10.5.3
